ivy/functional/ivy/manipulation.py

```diff
@@ -541,15 +541,103 @@ def squeeze(
         an output array having the same data type and elements as ``x``.
 
 
-    Examples
-    --------
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` input:
+
     >>> x = ivy.array([[[0, 1], [2, 3]]])
     >>> print(x.shape)
     (1, 2, 2)
 
     >>> print(ivy.squeeze(x, axis=0).shape)
     (2, 2)
 
+    >>> print(ivy.squeeze(x).shape)
+    (2, 2)
+
+    >>> x = ivy.array([[[[1, 2, 3]], [[4, 5, 6]]]])
+    >>> print(x.shape)
+    (1, 2, 1, 3)
+
+    >>> print(ivy.squeeze(x, axis=2).shape)
+    (1, 2, 3)
+
+    >>> x = ivy.array([[[0], [1], [2]]])
+    >>> print(x.shape)
+    (1, 3, 1)
+
+    >>> print(ivy.squeeze(x))
+    [0, 1, 2]
+
+    >>> print(ivy.squeeze(x, axis=0))
+    [[0], [1], [2]]
+
+    >>> print(ivy.squeeze(x, axis=2))
+    [[0, 1, 2]]
+
+    >>> print(ivy.squeeze(x, axis=(0, 2)))
+    [0, 1, 2]
+
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([0, 1, 2])
+    >>> print(ivy.squeeze(x))
+    ivy.array([0, 1, 2])
+
+    >>> x = ivy.native_array([[[3]]])
+    >>> print(x.shape)
+    (1, 1, 1)
+
+    >>> print(ivy.squeeze(x, 2))
+    ivy.array([[3]])
+
+    >>> x = ivy.native_array(0)
+    >>> print(x.shape)
+    ()
+
+    >>> print(ivy.squeeze(x, 0))
+    ivy.array(0)
+
+    With :code:`ivy.Container` input:
+
+    >>> x = ivy.Container(a=ivy.array([0., 1., 2.]), \
+                          b=ivy.array([3., 4., 5.]))
+    >>> y = ivy.squeeze(x)
+    >>> print(y)
+    {
+        a: ivy.array([0., 1., 2.]),
+        b: ivy.array([3., 4., 5.])
+    }
+
```

Comment:
Hi, these examples look great, just that can you copy and paste them to ivy.Array.squeeze and ivy.Container.squeeze according to [array instance method and container instance method example](https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html#docstring-examples). Should be ready to merge then. Thanks.

Reply:
Hi @MarShaikh. As you said, I have updated the instance methods of array and container. Since I do not have permission to merge, so could you please review and merge? Thank you in advance.